hardhat.config.js

```diff
@@ -0,0 +1,16 @@
+require('@nomiclabs/hardhat-ethers')
+require('dotenv').config()
+
+module.exports = {
+  defaultNetwork: 'hardhat',
+  networks: {
+    hardhat: {
+      blockGasLimit: 150_000_000,
+      forking: {
+        enabled: true,
+        url: `${process.env.ARCHIVE_NODE_RPC}`,
+        blockNumber: 12972642,
```

Comment:
hmm...is this for predictable test results?? wonder if there'd be a way to test things without needing exact numbers...haven't thought it through yet though

Reply:
Actually this was copy pasted over too and isn't needed. 

The tests already don't require exact numbers. Currently I check the users balance before and after the swap

e.g. For example for an exactIn swap of 100, i check the user has 100 less of tokenIn after the swap. I don't check tokenOut, but thinking more about it- i think it would be valuable to check token out is within slippage percentage of the quote. Ill add some more checks